app/project_type/git.py

```diff
@@ -77,7 +58,7 @@ def get_timing_file_directory(url):
     # pylint: disable=redefined-builtin
     # Disable "redefined-builtin" because renaming the "hash" parameter would be a breaking change.
     def __init__(self, url, build_project_directory='', project_directory='', remote='origin', branch='master',
```

Comment:
This might be out of scope for the PR, or maybe not--I think it's time we rename the 'branch' argument to 'ref'.